components/src/core/channel-value/channel-value.tsx

```diff
@@ -81,13 +91,16 @@ export const ChannelValue: React.FC<ChannelValueProps> = (props) => {
         iconSize,
         color,
         units,
+        unitSpace = 'auto',
         prefix = false,
         styles = {},
         style,
         theme: themeOverride,
         ...viewProps
     } = props;
     const theme = useTheme(themeOverride);
+    const prefixUnitWhitelist = ['$'];
+    const suffixUnitWhitelist = ['%', '℉', '°F', '℃', '°C', '°'];
```

Comment:
Can take these two out of the component, so that they do not get re-initialized every render

Reply:
good catch